### ISSUE TITLE
Support TNEF

### ISSUE THREAD

**Issue by jbremer** [jbremer/sflock#9]
See also https://sourceforge.net/p/ytnef/tnef-samples/1/, https://github.com/Yeraze/ytnef, and most importantly https://github.com/koodaamo/tnefparse.

**Comment by idiom** [comment]
Yeah, we could  switch the code in the .msg unpacker to use tnefparse instead of parsing the OLE streams directly. I can take a look at this if you want.

**Comment by jbremer** [comment]
Eh, that would also be a possibility ;-) If you have some spare time that'd be great!
